engine/src/main/java/com/redhat/cloud/notifications/processors/email/EmailSender.java

```diff
@@ -77,6 +78,38 @@ public void init() {
         bopApiToken = LineBreakCleaner.clean(bopApiToken);
     }
 
+    public void sendEmail(Set<User> user, Event event, TemplateInstance subject, TemplateInstance body, boolean persistHistory) {
```

Comment:
Nitpicky, but I'd vote for renaming `user` to `users`

Reply:
Sure